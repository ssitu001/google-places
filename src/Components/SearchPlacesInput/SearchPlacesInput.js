import React from 'react';

const SearchPlacesInput = ({handleSubmit, handleChange}) => {

  return (
    <div className="row">
      <div className="col-md-12">
        <form className="form-inline" onSubmit={handleSubmit}>
          <input type="text"
            className="form-control input-md"
            onChange={handleChange}
            id="place"
            placeholder="Search"
            required/>
        </form>
      </div>
    </div>
  )
}

export default SearchPlacesInput;