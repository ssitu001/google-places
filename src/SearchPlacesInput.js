import React from 'react';
import { ButtonGroup, Button, Form, FormGroup, FormControl, ControlLabel } from 'react-bootstrap';

const SearchPlacesInput = ({handleSubmit, handleChange}) => {

  return (
    <Form inline onSubmit={handleSubmit}>
      <FormGroup>
        <FormControl 
          type="text"
          placeholder="Search Place Here..."
          onChange={handleChange}
          required
        />
      </FormGroup>
    {' '}
      <Button type="submit">
        Search
      </Button>
    </Form>
  )
}

export default SearchPlacesInput;