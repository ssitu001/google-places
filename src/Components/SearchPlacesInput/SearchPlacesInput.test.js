import React from 'react';
import toJson from 'enzyme-to-json';

import SearchPlacesInput from './SearchPlacesInput';

test('renders SearchPlacesInput component as expected', () => {
  const component = shallow(<SearchPlacesInput />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});