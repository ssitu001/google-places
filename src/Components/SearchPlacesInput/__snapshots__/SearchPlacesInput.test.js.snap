// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchPlacesInput component as expected 1`] = `
<Form
  bsClass="form"
  componentClass="form"
  horizontal={false}
  inline={true}
>
  <FormGroup
    bsClass="form-group"
  >
    <FormControl
      bsClass="form-control"
      componentClass="input"
      placeholder="Search"
      required={true}
      type="text"
    />
  </FormGroup>
   
</Form>
`;
